scene/register_scene_types.cpp

```diff
@@ -397,12 +395,13 @@ void register_scene_types() {
 
 	/* REGISTER 3D */
 
-	// Needed even with _3D_DISABLED as used in animation code.
+#ifndef _3D_DISABLED
```

Comment:
Those no longer need to be separate then, they can go with the rest of the 3D stuff.
The `/* REGISTER 3D */` comment doesn't really match the Animation-related stuff registered here so the comments might need tweaking (moving 3D down, adding `/* REGISTER ANIMATION */` I guess.